webapp/auth_server.py

```diff
@@ -65,27 +64,37 @@ def signup():
     return render_template("signup.html")
 
 
-def add_user(id, name, group, join):
+def add_user(email, id, name, group, join):
```

Comment:
we might want to init resources -> water, food, metal for non-joining users. Maybe we can start with 20 units each?